lib/bump/pull_request_creator.rb

```diff
@@ -120,7 +120,8 @@ def default_branch
     end
 
     def new_branch_name
-      "bump_#{dependency.name}_to_#{dependency.version}"
+      path = ["bump", dependency.language, files.first.directory].reject(&:nil?)
```

Comment:
I think you might be thinking of `Hash#compact` - `Array#compact` is in core: https://ruby-doc.org/core-2.3.0/Array.html#method-i-compact